package.json

```diff
@@ -8,11 +8,13 @@
     "test": "npm run test:lint && npm run test:unit",
     "test:lint": "eslint . --ext jsx,js",
     "test:unit": "CSRF=false jest",
-    "build": "npm run build:css && npm run build:css:pdf",
-    "prebuild:css": "mkdir -p ./pages/common/dist/css",
-    "build:css": "npm-sass ./pages/common/assets/sass/style.scss > ./pages/common/dist/css/app.css",
-    "prebuild:css:pdf": "mkdir -p ./pages/project-version/pdf/css",
-    "build:css:pdf": "node-sass ./pages/project-version/pdf/sass -o ./pages/project-version/pdf/css",
+    "prebuild": "mkdir -p ./pages/common/dist/css/pdf",
+    "build": "npm run build:css",
+    "build:css": "npm run build:css:app && npm run build:css:pdf && npm run build:css:pdf:project",
+    "build:css:app": "npm-sass ./pages/common/assets/sass/style.scss > ./pages/common/dist/css/app.css",
+    "build:css:pdf": "node-sass ./pages/common/assets/sass/pdf -o ./pages/common/dist/css/pdf",
+    "prebuild:css:pdf:project": "mkdir -p ./pages/project-version/pdf/css",
+    "build:css:pdf:project": "node-sass ./pages/project-version/pdf/sass -o ./pages/project-version/pdf/css",
```

Comment:
Once this is merged I'll do another ticket to make projects use the new generic code, and remove the two `pdf:project` lines.

Reply:
it would be better to sort them out before merging IMO - there's gunna be a _lot_ of repeated stuff otherwise

Reply:
This is resolved in another open pages PR that migrates the project pdfs to use the generic code.

Reply:
https://github.com/UKHomeOffice/asl-pages/pull/528/files#diff-b9cfc7f2cdf78a7f4b91a753d10865a2